area_detector_handlers/handlers.py

```diff
@@ -154,6 +154,8 @@ def __call__(self, point_number):
                 raise IOError(H5PY_KEYERROR_IOERROR_MSG) from error
         start = point_number * self._fpp
         stop = (point_number + 1) * self._fpp
+        if len(self._dataset[start:stop]) == 0:
```

Comment:
A minor suggestion: this is more idiomatic Python, considered a little more readable.
```suggestion
        if not len(self._dataset[start:stop]):
```